blueprints/hpc-cluster-small.yaml

```diff
@@ -28,6 +28,17 @@ deployment_groups:
         id: network1
         use: [project1]
 
+      - source: ./modules/billing
+        kind: terraform
+        id: billing
+        settings:
+          display_name: example-alert"
+          amount: 1000
+          currency_code: USD
+          threshold_rules: [10]
```

Comment:
1. I think that the `threshold` values are `0.25`, `0.5`, `0.9` and `1.0`.
2. This might be a variable.